### PR TITLE
add sgc testnet

### PR DESCRIPTION
**Pull request by Zombieliu**
hello,

this is to add again the Sgc  testnet,

Thanks!

### CODE REVIEW

File: packages/apps-config/src/api/spec/index.ts [hunk 1]
```diff
@@ -54,6 +54,8 @@ import trustbase from './trustbase';
 import uniarts from './uniarts';
 import zenlink from './zenlink';
 import zero from './zero';
+import sgc from './sgc';
```
Comment (jacogr) [inline packages/apps-config/src/api/spec/index.ts:57]: Please add this alphabetical, as well as below where it is used. (It really is simpler to maintain, there are a lot)

File: packages/apps-config/src/endpoints/testing.ts [hunk 2]
```diff
@@ -231,6 +231,13 @@ export function createTesting (t: TFunction): LinkOption[] {
       providers: {
         ZERO: 'wss://alphaville-0.zero.io'
       }
+    },
+    {
+      info: 'sgc',
+      text: t('rpc.sgc', 'Sgc', { ns: 'apps-config' }),
```
Comment (jacogr) [inline packages/apps-config/src/endpoints/testing.ts:237]: Also adjust alphabetically.

File: packages/apps-config/src/ui/colors.ts [hunk 3]
```diff
@@ -47,6 +47,7 @@ const chainUniarts = '#b39ef7';
 const chainIdavoll = '#ff43ff';
 const chainSubDAO = 'linear-gradient(50deg, #F20092 0%, #FF4D5D 100%)';
 const chainTrustBase = '#ff43aa';
+const chainSgc = 'linear-gradient(45deg, #0099F7 0%, #F11712 100%)';
```
Comment (jacogr) [inline packages/apps-config/src/ui/colors.ts:50]: Same here ...

I have been cleaning up as I go through, there is still a bit that is not in the correct order depending on the file.

**Review by jacogr — COMMENTED** [review]
Looks good. 

Please just apply alphabetical ordering where it is added in all files. (I do know in some places they are not - just because I let things slip in and without it is is really difficult to maintain. Have been cleaning up, but have not gotten to all as of yet)

File: packages/apps-config/src/endpoints/testing.ts (hunk 2 above)
Comment (Zombieliu) [reply packages/apps-config/src/endpoints/testing.ts]: OK the latest change is third add sgc network Thank you very much!!!

File: packages/apps-config/src/ui/colors.ts (hunk 3 above)
Comment (Zombieliu) [reply packages/apps-config/src/ui/colors.ts]: OK the latest change is third add sgc network Thank you very much!!!

**Review by Zombieliu — COMMENTED** [review]
OK the latest change is third add sgc network Thank you very much!!!

**Comment by Zombieliu** [comment]
the latest change is third add sgc network Thank you very much!!!

**Review by Zombieliu — COMMENTED** [review]
！！！！！！ the latest add !!! finally

File: packages/apps-config/src/ui/logos/index.ts [hunk 4]
```diff
@@ -58,6 +58,7 @@ import nodePolkadex from './nodes/polkadex.svg';
 import nodePolkadot from './nodes/polkadot-circle.svg';
 import nodePolkadotJs from './nodes/polkadot-js.svg';
 import nodeRobonomics from './nodes/robonomics.svg';
+import nodeSgc from './nodes/Sgc.svg';
```
Comment (jacogr) [inline packages/apps-config/src/ui/logos/index.ts:61]:
```suggestion
import nodeSgc from './nodes/sgc.svg';
```

Case sensitivity.

File: packages/apps-config/src/ui/logos/index.ts [hunk 5]
```diff
@@ -68,6 +69,8 @@ import nodeTrustBase from './nodes/trustbase.png';
 import nodeUniarts from './nodes/uniarts.png';
 import nodeZenlink from './nodes/zenlink.svg';
 import nodeZero from './nodes/zero.svg';
+
```
Comment (jacogr) [inline packages/apps-config/src/ui/logos/index.ts:72]:
```suggestion
```

**Review by jacogr — COMMENTED** [review]
2 small comments, the case-sensitive one made the CI build fail.

Apart from that, thank you, once updated happy to merge. (You can just do it via GH, the quickest, add to batch, commit)

File: packages/apps-config/src/ui/logos/index.ts (hunk 4 above)
Comment (Zombieliu) [reply packages/apps-config/src/ui/logos/index.ts]: i see!!! Blame me for not being careful ! Thanks!

File: packages/apps-config/src/ui/logos/index.ts (hunk 5 above)
Comment (Zombieliu) [reply packages/apps-config/src/ui/logos/index.ts]: i see!!! Blame me for not being careful ! Thanks!

File: packages/apps-config/src/api/spec/index.ts [hunk 6]
```diff
@@ -114,7 +118,7 @@ const spec: Record<string, OverrideBundleDefinition> = {
   ternoa,
   trustbase,
   uniarts,
-  zenlink
+  zenlink,
```
Comment (jacogr) [inline packages/apps-config/src/api/spec/index.ts:121]:
```suggestion
  zenlink
```

File: packages/apps-config/src/api/spec/index.ts [hunk 7]
```diff
@@ -55,6 +56,8 @@ import uniarts from './uniarts';
 import zenlink from './zenlink';
 import zero from './zero';
 
+
+
```
Comment (jacogr) [inline packages/apps-config/src/api/spec/index.ts:60]:
```suggestion
```

File: packages/apps-config/src/ui/logos/index.ts [hunk 8]
```diff
@@ -249,6 +251,7 @@ export const namedLogos: Record<string, unknown> = {
   rococoTrick: chainRococoTrick,
   rococoTrustBase: nodeTrustBase,
   rococoZenlink: nodeZenlink,
+  sgc:nodeSgc,
```
Comment (jacogr) [inline packages/apps-config/src/ui/logos/index.ts:254]:
```suggestion
  sgc: nodeSgc,
```

**Review by jacogr — COMMENTED** [review]
Last couple of complains for CI, e.g. https://github.com/polkadot-js/apps/pull/4681/checks?check_run_id=1918899011#step:3:113

**Review by jacogr — APPROVED** [review]
Thank you!

**Comment by Zombieliu** [comment]
amazing !!! i git push 12 times commit ! ahahahahhahah my god

**Comment by jacogr** [comment]
The linting is fixable by just running `yarn lint` locally - it does autofix whatever it can. 

(It does not run on CI since it _may_ adjust things like types, where the ignore header is not available, and that would break the operation - it will look good in the PR, but the committed code would be broken)

**Pull request merged**